torchaudio/csrc/utils.cpp

```diff
@@ -30,12 +34,21 @@ bool is_ffmpeg_available() {
 #endif
 }
 
+int64_t cuda_version() {
+#ifdef USE_CUDA
+  return CUDA_VERSION;
+#else
+  return -1;
```

Comment:
PyTorch seems to return `None` in case it's not present. How about doing the same here?
I think that the following would do.

```c++
c10::optional<int64_t> cuda_version() {
#ifdef USE_CUDA
   return CUDA_VERSION;
 #else
   return {};
} 
```